react/features/base/connection/actions.native.js

```diff
@@ -310,7 +313,7 @@ function _constructOptions(state) {
         // XXX The Jitsi Meet deployments require the room argument to be in
         // lower case at the time of this writing but, unfortunately, they do
         // not ignore case themselves.
-        room && (bosh += `?room=${room.toLowerCase()}`);
+        room && (bosh += `?room=${getBackendSafeRoomName(room)}`);
```

Comment:
Can you drop the comment above the line now? There is no point in stating the obvious IMHO.